_plugins/replace-regex.rb

```diff
@@ -0,0 +1,10 @@
+module Jekyll
+	module RegexFilter
+		def replace_regex(input, regex_string, replace_string)
```

Comment:
I think it was in the cloudcannon theme that I started with on this branch, I can remove it.